Library/Homebrew/extend/os/mac/extend/ENV/super.rb

```diff
@@ -64,7 +67,10 @@ def homebrew_extra_library_paths
 
   def homebrew_extra_cmake_include_paths
     paths = []
-    paths << "#{self["HOMEBREW_SDKROOT"]}/usr/include/libxml2" unless deps.any? { |d| d.name == "libxml2" }
+    unless deps.any? { |d| d.name == "libxml2" } ||
+           Pathname("#{self["HOMEBREW_SDKROOT"]}/usr/include/libxml").directory?
```

Comment:
```suggestion
    if deps.none? { |d| d.name == "libxml2" } &&
           !Pathname("#{self["HOMEBREW_SDKROOT"]}/usr/include/libxml").directory?
```

Might be worth extracting to a method given it's used twice.